frontend/src/workers/xc-score-worker.ts

```diff
@@ -0,0 +1,42 @@
+import { solver, scoringRules } from 'igc-xc-score';
+import { IGCFile } from 'igc-parser';
+
+/* The joy of JS multithreading */
+function filterFunc(o: any): object {
```

Comment:
Could we drop that an be explicit in what we pass instead of using `filterFunc(something)` ?